anvil/core/src/eth/transaction/mod.rs

```diff
@@ -1044,6 +1044,8 @@ pub struct TransactionInfo {
     pub logs: Vec<Log>,
     pub logs_bloom: Bloom,
     pub traces: Vec<CallTraceNode>,
+    pub exit: Return,
```

Comment:
let's name this status, which should be consistent with how we currently name fields for the `Return` type
```suggestion
    pub status: Return,
```

Reply:
I'll gladly rename it, but I just want to point out it is named `exit` for `ExecutedTransaction`, which is why I kept this name: https://github.com/foundry-rs/foundry/blob/6fc06c5539efb86b0204331f8a5749a60390389a/anvil/src/eth/backend/executor.rs#L27-L34

Reply:
oh, good point, then let's stick `exit` here!